core/src/classes/commonModel.ts

```diff
@@ -45,7 +45,7 @@ export abstract class CommonModel<T> extends Model {
   }
 
   @BeforeBulkCreate
-  static validateIds(instances) {
+  static validateIds(instances: CommonModel<any>[]) {
```

Comment:
I think the use of Generics would allow this to function a lot nicer. Not necessary for this PR however.